docs/SOPs/Triage/GDPR_Guidelines.md

```diff
@@ -38,6 +69,6 @@ When wrangling any kind of data or metadata related to living donors, wranglers
 When wrangling **contributor led** publications, wranglers cannot recommend or suggest the contributors to archive their data in an open database if living donors are involved. Wranglers are also not allowed to state to the contributors that they could take the data and metadata from an open database once it's published there, as it might lead the contributors to make their data publicly available. **HCA recommendation when dealing with living donors should always be managed access**
 
 ### Metadata
-When wrangling from already **published projects**, wranglers should **never ask for extra donor metadata** if it involves living donors. Wranglers are only allowed to take the information that the authors have made already of public domain. Moreover, to prevent low *k-anonymity* [^1] levels, wranglers should **avoid wrangling donor metadata when the project includes less than 5 living donors**. 
+When wrangling from already **published projects**, wranglers should **never ask for extra donor metadata** if it involves living donors. Wranglers are allowed to take the information that the authors have made already of **public domain** or the authors provide more metadata that what's already publicly available but also provide **written consent** for its public release then the additional metadata can be wrangled as open access. ~~Moreover, to prevent low *k-anonymity* [^1] levels, wranglers should **avoid wrangling donor metadata when the project includes less than 5 living donors**. ~~
```

Comment:
```suggestion
When wrangling from already **published projects**, wranglers should **never ask for extra donor metadata** if it involves living donors. Wranglers are allowed to take the information that the authors have made already of **public domain**. Wranlger may include previously unreleased metadata provided by the author only if the author gave  **written consent** for its public release as open access. ~~Moreover, to prevent low *k-anonymity* [^1] levels, wranglers should **avoid wrangling donor metadata when the project includes less than 5 living donors**. ~~
```